release/models/system/openconfig-system-logging.yang

```diff
@@ -473,6 +479,142 @@ module openconfig-system-logging {
     }
   }
 
+  grouping logging-file-config {
+    description
+      "Configuration data for logfile";
+
+    leaf filename-prefix {
+      type string {
+        length 0..255;
+      }
+      description
+        "filename prefix. It is expected that implementation may append to it timestamp, serial-number or other type of identifier";
+    }
+
+    leaf path {
+      type string {
+        length 0..255;
+      }
+      description
+        "path to the folder where logfile is stored. May include drive identifier is implementation requires";
+    }
+
+    leaf count {
```

Comment:
Minor comment, but consider naming this leaf `rotate`, similar to the linux logrotate syntax.